server/src/http/refresher/feature_refresher.rs

```diff
@@ -249,6 +254,33 @@ impl FeatureRefresher {
         }
     }
 
+    pub(crate) async fn delta_events_for_filter(
```

Comment:
I think this is more or less a copy paste of `features_for_filter`. That's sometimes okay but I think in this case the subsumption and strict mode are fairly important system constraints that I think should change in lock step if they ever need to.

Is it tremendously difficult to get them to share the important stuff?

Reply:
Yes